packages/tools/src/tools/displayTools/Contour/contourDisplay.ts

```diff
@@ -213,20 +213,20 @@ function _checkContourGeometryMatchViewport(
   }
 
   const contours = geometryData.contours;
-  const points = contours[0].points;
-  const point1 = points[0];
-  const point2 = points[1];
-  const point3 = points[2];
-
-  // get the normal from two points
-  let normal = vec3.cross(
-    vec3.create(),
-    vec3.sub(vec3.create(), point2, point1),
-    vec3.sub(vec3.create(), point3, point1)
-  );
-  normal = vec3.normalize(vec3.create(), normal);
-  const dotProduct = vec3.dot(normal, viewportNormal);
-  return Math.abs(dotProduct) > 0.9;
+  const { points } = contours[0];
```

Comment:
The old code was invalid if the first three points happened to be colinear.  The new version is much closer to correct is it would require 25 colinear points (or all the points for less than 25 points) to fail to be orthogonal with respect to point 0.
It also works for 0,1 and 2 point contours.